### PR TITLE
Sequelize models (User, Cart, Reviews)

### PR DESCRIPTION
**Pull request by emmacollins**
this needs review!

### CODE REVIEW

File: db/models/index.js [hunk 1]
```diff
@@ -4,9 +4,44 @@
 // so any other part of the application could call sequelize.model('User')
 // to get access to the User model.
 
-const User = require('./user')
-const OAuth = require('./oauth')
+const User = require('./user');
+const OAuth = require('./oauth');
+const Review = require('./review');
+//const Product = require('./product');
```
Comment (clairefreehafer) [inline db/models/index.js]: comments that can be removed ?

File: db/index.js [hunk 2]
```diff
@@ -32,19 +32,19 @@ function sync(force=app.isTesting, retries=0, maxRetries=5) {
       // Don't do this auto-create nonsense in prod, or
       // if we've retried too many times.
       if (app.isProduction || retries > maxRetries) {
-        console.error(chalk.red(`********** database error ***********`))
-        console.error(chalk.red(`    Couldn't connect to ${url}`))
+        console.error(chalk.red(`********** database error ***********`));
```
Comment (elliette) [inline db/index.js]: We actually don't need these semicolons- Emily mentioned that is Bones convention to not use semicolons.

File: db/models/index.js (hunk 1 above)
Comment (clairefreehafer) [inline db/models/index.js]: just kidding those are our modules

File: db/models/user.js [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
   // We support oauth, so users may or may not have passwords.
   password_digest: Sequelize.STRING, // This column stores the hashed password in the DB, via the beforeCreate/beforeUpdate hooks
-	password: Sequelize.VIRTUAL // Note that this is a virtual, and not actually stored in DB
+	password: Sequelize.VIRTUAL, // Note that this is a virtual, and not actually stored in DB
+  isAdmin: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  }
 }, {
-	indexes: [{fields: ['email'], unique: true,}],
```
Comment (elliette) [inline db/models/user.js]: Emily also mentioned that trailing commas are a Bones convention. I think they look silly too.

**Pull request merged**